docs/providers.md

```diff
@@ -33,7 +33,7 @@ We encourage you to create your own provider using the [Provider](../specs/provi
 | name | version | build status |
 | :--- | :------ | :----------- |
 | [Cloudant](https://github.com/cloudant/koop-cloudant) | [![npm](https://img.shields.io/npm/v/koop-cloudant.svg?style=flat-square)](https://www.npmjs.com/package/koop-cloudant) | [![travis](https://img.shields.io/travis/cloudant/koop-cloudant.svg?style=flat-square)](https://travis-ci.org/cloudant/koop-cloudant) |
-| [GeoJSON file](https://github.com/jseppi/koop-geojson-file) | [![npm](https://img.shields.io/npm/v/koop-geojson-file.svg?style=flat-square)](https://www.npmjs.com/package/koop-geojson-file) | [![travis](https://img.shields.io/travis/jseppi/koop-geojson-file.svg?style=flat-square)](https://travis-ci.org/jseppi/koop-geojson-file) |
+| [GeoJSON file](https://github.com/koopjs/koop-provider-geojson) | [![npm](https://img.shields.io/npm/v/koop-geojson-file.svg?style=flat-square)](https://www.npmjs.com/package/koop-geojson-file) | [![travis](https://img.shields.io/travis/jseppi/koop-geojson-file.svg?style=flat-square)](https://travis-ci.org/jseppi/koop-geojson-file) |
```

Comment:
if you're going to point at `koop-provider-geojson`, we should get rid of the associated `npm` and `travis` links.

Reply:
Makes a lot of sense, and I'll look for where I can find that since the other repo isn't tracking the same things.

Reply:
yeah, since that provider hasn't been published on npm and doesn't have continuous integration tests running in travis, we should just remove those badges entirely.